### ISSUE TITLE
Disk size limits do not work as expected

### ISSUE THREAD

**Issue by Derkades** [moosefs/moosefs-docker-cluster#16]
When creating four chunk servers with a single disk (but actually all on the same host disk), environment variable `SIZE=10` one expects to get four chunk servers that show 0GB / 10GB (0%) used. However, the web interface shows 10GB / 10GB (100%) used. Creating files fails with an out of space error.

![image](https://user-images.githubusercontent.com/15892014/99195976-96dcfd00-2789-11eb-962b-8ff4c518d4c0.png)

When removing the size limit the cause of this issue becomes immediately obvious:
![image](https://user-images.githubusercontent.com/15892014/99195884-e0791800-2788-11eb-933a-77ccab97d7a3.png)
To calculate 'used' space moosefs uses the drive's used space instead of the size of files actually stored in the directory. This makes a lot of sense due to performance and inconsistency issues when counting individual files, but it makes the `SIZE` option a lot less useful.

It took me quite a while to figure out why all my disks showed as full, if there is no good fix for this at least you should document this in the README or manual in my opinion.

**Comment by Derkades** [comment]
It appears that as soon as one object is uploaded it calculates used space correctly
![image](https://user-images.githubusercontent.com/15892014/99246287-58852380-2805-11eb-9e28-fb726399159f.png)

**Comment by rocinante-sys** [comment]
my test setup does not want to store anything. it says no space on device :)

**Comment by Derkades** [comment]
@rocinante-sys Are you using storage quotas (`SIZE` environment variable)?

**Comment by rocinante-sys** [comment]
@Derkades yes, I saw in first post that is the issue. It works if I leave it out :+1:
